src/Tracing/PropagationContext.php

```diff
@@ -65,6 +67,14 @@ public function toTraceparent(): string
         return sprintf('%s-%s', (string) $this->traceId, (string) $this->spanId);
     }
 
+    /**
+     * Returns a string that can be used for the W3C `traceparent` header & meta tag.
+     */
+    public function toW3CTraceparent(): string
+    {
+        return sprintf('00-%s-%s', (string) $this->traceId, (string) $this->spanId);
```

Comment:
https://www.w3.org/TR/trace-context/#version-format

Hardcoding the version to `00` seems to be okay.